### DISCUSSION TITLE
ProtoPediaの登録方法　14時半までに登録

### DISCUSSION THREAD

**Discussion by banboo1118** [1ft-seabass/ms-azure-hackathon-league-2021#28] [Show and tell]
ProtoPediaはハッカソンなどで生まれた作品を登録するWebメディアです。
ハッカソンの指定時間（今回の場合は14:30まで）までに作品の情報を「公開」で登録ください。
登録するとURLが発行されますので、今回のハッカソンの成果物として拡散する際など活用ください。
※審査員にも共有いたします。

▼登録例
https://protopedia.net/prototype/2697
▼登録方法
https://protopedia.gitbook.io/helpcenter/registration

**Comment by TKTK360** [comment]
【TEAM】えがお 1360％ （Bチーム）
【タイトル】パーソナルAI アシスタント ～ 皆の笑顔を取り戻せ！！ ～
【URL】https://protopedia.net/prototype/2810
　※補足：デモ動画は後程アップ予定です。

**Comment by aono-tomo** [comment]
【TEAM】ヒゲ野郎 （Dチーム）
【タイトル】オンラインプリクラ
【URL】https://protopedia.net/prototype/2811

**Comment by naoya5614** [comment]
【チーム名】BUREIKOU
【作品タイトル】BUREIKOU
【URL】https://protopedia.net/prototype/2812
※未完です。

**Comment by d-nakajima** [comment]
【チーム名】ソーシャルレジデンス福生
【作品タイトル】得笑い
【URL】https://protopedia.net/prototype/2814  
※一旦作品だけ作成しました

**Comment by uecken** [comment]
【チーム名】OH YEAH!
【作品タイトル】LASHBOARD
【URL】https://protopedia.net/prototype/2815

(チーム名、変わってます)

**Comment by aoki0satoshi** [comment]
【TEAM】4ビット （Hチーム）
【タイトル】トーク「うん」
【URL】https://protopedia.net/prototype/2816
　※補足：写真・デモ動画は後程アップ予定です。

**Comment by ksera332** [comment]
【TEAM】チームへんじにあ （Uチーム）
【タイトル】食材デリバリーアプリ
【URL】https://protopedia.net/prototype/2816
https://protopedia.net/prototype/2830

**Comment by putrax57** [comment]
【TEAM】チキンライス （Sチーム）
【タイトル】話題を促進する笑クマ
【URL】https://protopedia.net/prototype/2829

**Comment by Haruhiko04** [comment]
【TEAM】電波教室 （Rチーム）
【タイトル】Virtual Tutur
【URL】https://protopedia.net/prototype/2832
